### PR TITLE
Migrate to Fedora Messaging

### PR DESCRIPTION
**Pull request by abompard**
Use the Fedora Messaging library, and migrate to stompest instead of the stomper-based moksha plugin.

Also, the app is Python3 now.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,24 +1,35 @@
...
+#!/usr/bin/env python
+
+from setuptools import setup, find_packages
+
 
 setup(
     name='bugzilla2fedmsg',
-    version='0.3.1',
-    description='Consume BZ messages over STOMP and republish to fedmsg',
+    version='0.4.0',
```
Comment (pypingou) [inline setup.py]: Would it be worth bumping the major version since this isn't quite backward compatible?

**Review by Zlopez — APPROVED** [review]

**Comment by AdamWill** [comment]
So, I just wrote a big PR to fix the processing of Bugzilla 5 messages:

https://github.com/fedora-infra/bugzilla2fedmsg/pull/13

unfortunately it will conflict with this. The conflict shouldn't be too hard to resolve, though, once one or the other gets pushed we can adjust the other one.

**Comment by abompard** [comment]
OK Adam, I'll bump the version in this one as suggested by @pypingou  and then merge it. If you need help to resolve the conflicts in your PR, I can totally help.

**Comment by AdamWill** [comment]
I actually thought it'd be easier to merge them the other way around :( This one would probably only have one or two conflicting commits on top of mine, but I think *all* the commits in mine will conflict now this has changed where the affected code lives :( Oh well, I'll work through it.

**Comment by AdamWill** [comment]
Is this 'stompest' thing not even packaged yet?

**Comment by AdamWill** [comment]
Note this was missing a schema:

https://fedora-messaging.readthedocs.io/en/latest/messages.html#schema

We should add one; this should probably be done after my bz5 fixes PR is merged, because the schema would look different with and without it.

**Pull request merged**